src/main/kotlin/de/bund/digitalservice/useid/widget/WidgetController.kt

```diff
@@ -106,11 +107,16 @@ class WidgetController(
 
         return Rendering
             .view(WIDGET_PAGE)
-            .model(defaultViewHeaderConfig + widgetViewConfig)
+            .model(defaultViewHeaderConfig + widgetViewFallbackConfig)
             .status(HttpStatus.OK)
             .build()
     }
 
+    private fun publishMatomoEvent(category: String, action: String, name: String) {
```

Comment:
Category is always "Widget" since it's the widget controller. So this could be inlined in the method and the method could have a name like `publishWidgetMatomoEvent`.